draft-ietf-httpbis-digest-headers.md

```diff
@@ -630,7 +633,7 @@ Specification document(s):  {{want-fields}} of this document
 
 --- back
 
-# Resource Representation and Representation-Data {#resource-representation}
+# Resource Representation and Representation Data {#resource-representation}
```

Comment:
This is a weird title now that we removed the term `Representation-Data`. Maybe there's a better alternative?

Reply:
We removed `representation-data-digest`. Representation data is the concept defined in SEMANTICS.